### DISCUSSION TITLE
i-panel not rendering

### DISCUSSION THREAD

**Discussion by rdlepine** [primefaces/community#2600] [PrimeNG]
Using angular 20 and primng 17.  When trying to use i-panel the component is not visible. It does take up space which i proved by adding width and height in the css file. Are there any other dependencies other that PanelModule?
Regards,
Ron

**Comment by rdlepine** [comment]
Sorry p-panel

**Comment by qwadrox** [comment]
@rdlepine Ensure that you have the required css dependencies. https://primeng.org/installation#styles
Both the primeng theme and core css file. 

Do you see any error on the console?
